sysconfig/cloudinit.go

```diff
@@ -743,6 +744,16 @@ func CloudInitStatus() (CloudInitState, error) {
 		return CloudInitDisabledPermanently, nil
 	}
 
+	// if it was explicitly disabled via the kernel commandline, then
+	// return special status for that, if commandline is unavailable,
+	// assume not disabled
```

Comment:
Nitpick, "assume not disabled" does not seem accurate, we do not assume this,  but rely on "cloud-init" file based checks to determine CloudInitState?